wpilibc/shared/include/Commands/Command.h

```diff
@@ -66,8 +66,7 @@ class Command : public ErrorBase, public NamedSendable, public ITableListener {
   bool IsInterruptible() const;
   void SetInterruptible(bool interruptible);
   bool DoesRequire(Subsystem* subsystem) const;
-  typedef std::set<Subsystem*> SubsystemSet;
-  SubsystemSet GetRequirements() const;
```

Comment:
With the current typedef name, we are stuck with using a set, so that just leaves the template type. The user needs to know what that type is to do anything useful with the set, so changing the typedef would still break code. I don't see any advantage here.